Update samples to 4.16 (#3734)
microsoft/BotBuilder-Samples: samples/csharp_dotnetcore/11.qnamaker/README.md

﻿# QnA Maker

> **Important note:** The QnA Maker service is being retired on the 31st of March, 2025. A newer version of the question and answering capability is now available as part of Azure Cognitive Service for Language. 
> For question answering capabilities within the Language Service, see question answering. Starting 1st October, 2022 you won’t be able to create new QnA Maker resources. 
> For information on migrating existing QnA Maker knowledge bases to question answering, consult the migration guide.
> For an updated version of this sample using the new service see the [CustomQABot sample](../12.customQABot)

Reviewer: General edit for this note as it appears in any of the READMEs:
```suggestion
**Important**: The QnA Maker service will retire on the 31st of March, 2025. A newer version of the question and answering capability is now available as part of Azure Cognitive Service for Language.
For question-and-answer capabilities within the Cognitive Services, see _question answering_. Starting 1st October, 2022 you won’t be able to create new QnA Maker resources. 
For information on migrating existing QnA Maker knowledge bases to question answering, consult the migration guide.
For an updated version of this sample that uses question answering, see the [CustomQABot sample](../12.customQABot).

```
